lib/hyperion/headers.rb

```diff
@@ -27,7 +27,8 @@ def short_mimetype(response_descriptor)
     end
 
     ContentTypes = [[:json, 'application/json'],
-                    [:protobuf, 'application/x-protobuf']]
+                    [:protobuf, 'application/x-protobuf'],
+                    [:form_data, 'application/x-www-form-urlencoded']]
```

Comment:
This feels like a hack to me. Does it make sense to be able to read a "format" we cannot write? This doesn't really solve the problem of modeling the receiving of multipart data. How would you pick out the multiple parts? If you need this to get unblocked, then I'll merge it, but please add some code comments indicating that `:form_data` is a hack to allow fake server to receive multipart requests. We can improve it later.